setup.cfg

```diff
@@ -27,6 +27,7 @@ install_requires =
     requests >= 2.26,<3
     vega >= 3.5,<4
     xarray >=0.19,<2022.4
+    matplotlib >=3.4,<4
```

Comment:
Two things:

1. If matplotlib is now part of `install_requires`, remove it from `test`

2. Order the dependencies alphabetically

Reply:
It should also be removed from `examples`

Reply:
addressed with [[a22470c](https://github.com/geo-engine/geoengine-python/pull/64/commits/a22470ca68167a05d47b588da7957a577d716618)]